docker/opa-policies/terraform.rego

```diff
@@ -92,7 +92,7 @@ resources[resource_type] = all {
     ]
 }
 
-other_resources[resource_type] := all {
```

Comment:
This should work with OPA v0.40.0, and is the preferred operator going forward.

Reply:
Ah, we should use `=` instead of `:=` in 0.40.0 and forward? 🤷‍♂️

Reply:
See https://github.com/open-policy-agent/opa/issues/4555

Reply:
Thank you @anderseknert!

@NissesSenap maybe we should just update the OPA version instead? Perhaps we should go through the code and make sure we have the same operator everywhere as well.

Reply:
Just to be clear — either will work as expected, so this was very much just a nit :) Mainly just surprised to see it going in this direction. But if compatibility with older versions of OPA is a concern, then that's definitely a valid reason.

Reply:
I will update the other PR so it includes a new version of opa and fix the syntax.